barretenberg/cpp/src/barretenberg/polynomials/pow.hpp

```diff
@@ -55,77 +43,80 @@ namespace barretenberg {
  *
  * We want to check that P(i)=0 for all i ∈ {0,1}^d. So we use Sumcheck over the polynomial
  * P'(X) = pow(X)⋅P(X).
- * The claimed sum is 0 and is equal to ∑_{i ∈ {0,1}^d} pow(i)⋅P(i) = ∑_{i ∈ {0,1}^d} ζ^{i}⋅P(i)
+ * The claimed sum is 0 and is equal to ∑_{i ∈ {0,1}^d} pow(i)⋅P(i).
  * If the Sumcheck passes, then with it must hold with high-probability that all P(i) are 0.
  *
- * The trivial implementation using P'(X) directly would increase the degree of our combined relation by 1.
- * Instead, we exploit the special structure of pow to preserve the same degree.
- *
- * In each round l, the prover should compute the univariate polynomial for the relation defined by P'(X)
- * S'ˡ(X_l) = ∑_{ 0 ≤ i < 2^{d-l-1} } powˡᵢ( X_l ) Sˡᵢ( X_l ) .
- *        = ∑_{ 0 ≤ i < 2^{d-l-1} } [ ζ_{l+1}ⁱ⋅( (1−X_l) + X_l⋅ζ_l )⋅c_l ]⋅Sˡᵢ( X_l )
- *        = ( (1−X_l) + X_l⋅ζ_l ) ⋅ ∑_{ 0 ≤ i < 2^{d-l-1} } [ c_l ⋅ ζ_{l+1}ⁱ ⋅ Sˡᵢ( X_l ) ]
- *        = ( (1−X_l) + X_l⋅ζ_l ) ⋅ ∑_{ 0 ≤ i < 2^{d-l-1} } [ c_l ⋅ ζ_{l+1}ⁱ ⋅ Sˡᵢ( X_l ) ]
- *
- * If we define Tˡ( X_l ) := ∑_{0≤i<2ˡ} [ c_l ⋅ ζ_{l+1}ⁱ ⋅ Sˡᵢ( X_l ) ], then Tˡ has the same degree as the original Sˡ(
- * X_l ) for the relation P(X) and is only slightly more expensive to compute than Sˡ( X_l ). Moreover, given Tˡ( X_l ),
- * the verifier can evaluate S'ˡ( u_l ) by evaluating ( (1−u_l) + u_l⋅ζ_l )Tˡ( u_l ). When the verifier checks the
- * claimed sum, the procedure is modified as follows
- *
  * Init:
- * - σ_{ 0 } <-- 0 // Claimed Sumcheck sum
- * - c_{ 0 } <-- 1 // Partial evaluation constant, before any evaluation
- * - ζ_{ 0 } <-- ζ // Initial power of ζ
+ * - σ_0 <-- 0 // Claimed Sumcheck sum
```

Comment:
I think we want to keep these brances because htey will be needed to render latex in doxygen